src/plugins/data/public/utils/table_inspector_view/components/data_table.tsx

```diff
@@ -35,82 +35,123 @@ import { i18n } from '@kbn/i18n';
 
 import { DataDownloadOptions } from './download_options';
 import { DataViewRow, DataViewColumn } from '../types';
-import { TabularData } from '../../../../common/adapters/data/types';
 import { IUiSettingsClient } from '../../../../../../core/public';
+import {
+  Datatable,
+  DatatableColumn,
+} from '../../../../../expressions/common/expression_types/specs';
```

Comment:
nit: Since this is a `public` directory, should probably import from `src/plugins/expressions/public` instead of a specific `common` subdirectory